contact.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Contact Form</title>
+    <link rel="stylesheet" href="contact.css">
+</head>
+
+<body background="IMG_1693.jpg">
+
+
+
+    <div class="Biography">
+        <p>My philosophy!</p>
```

Comment:
This probably makes more sense as an html-header element (`h1` / `h2` / `h3` etc.)